srtcore/buffer.h

```diff
@@ -245,6 +254,9 @@ class CSndBuffer
 
 class CRcvBuffer
 {
+    typedef srt::sync::steady_clock::time_point time_point;
+    typedef srt::sync::steady_clock::duration duration;
```

Comment:
time_point_t?
duration_t?

Reply:
Although it is `time_point` in sync.h

Reply:
This is to improve cleanness in the header file - avoid spelling the full namespace name, whereas simultaneously avoiding conflicts and spreading the shortcut further than one class.